csharp/ql/src/semmle/code/csharp/dataflow/internal/DataFlowPrivate.qll

```diff
@@ -2194,6 +2206,20 @@ private class ReadStepConfiguration extends ControlFlowReachabilityConfiguration
 
 predicate readStep = readStepImpl/3;
 
+/**
```

Comment:
I don't understand the difference between `DataFlowType` and `Gvn::GvnType`. `Gvn::GvnType` has the following comment on it, which is pretty much the same as the one on `DataFlowType`.
```
A global value number for a type. Two types have the same GVN when they
are structurally equal modulo type parameters and identity conversions.
For example, `Func<T, int>` and `Func<S, int>` have the same GVN, but
`Func<T, int>` and `Func<string, int>` do not.
```

Reply:
`DataFlowType` is simply a subset of `Gvn::GvnType`; namely those that are the result of `NodeImpl::getDataFlowType()`. I decided to make the QL doc for `DataFlowType` a bit more data-flow centric, compared to the QL doc for the more generally applicable `Gvn::GvnType`.